uswds/uswds-site
# USWDS - Changelog: Migrate group 1 component data

## Summary
**Added data for group 3 components.** This adds release data to component changelogs from release `3.2.0 → 2.11.0`. 

## Related issue
Closes #1865

## Preview link
Preview links:
- [Accordion](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/accordion/)
- [Alert](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/alert/)
- [Banner](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/banner/)
- [Breadcrumb](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/breadcrumb/)
- [Button group](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/button-group/)
- [Button](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/button/)
- [Card](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/card/)
- [Character count](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/character-count/)
- [Checkbox](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/checkbox/)
- [Collection](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/collection/)
- [Combo-box](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/combo-box/)
- [Date picker](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/date-picker/)
- [Date range picker](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/date-range-picker/)
- [File input](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/file-input/)
- [Footer](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/footer/)
- [Header](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/header/)
- [Icon list](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/icon-list/)
- [Icon](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/icon/)
- [Identifier](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/identifier/)
- [Input prefix/suffix](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-group-1/components/input-prefix-suffix/)

[See all groups in spreadsheet 🔒](https://docs.google.com/spreadsheets/d/16W0IO8gTW3yv47Dy9sUBAkEAkvVzpOYQpg2R9C6CM5s/edit#gid=0)

## How to test
1. Visit component pages above
2. Click on "Latest updates" in the side nav, or scroll to bottom of page
3. Verify new **Latest updates** section is listed
4. Validate accuracy of data
   1. Release dates should be correct
   2. Versions should be correct
   3. Affected tags are relevant
   4. Breaking changes are documented correctly
   6. Descriptions are unique and free of typo's
   7. Links to PRs are correct
